drake_cmake_installed/README.md

```diff
@@ -2,24 +2,48 @@
 
 This uses the CMake `find_package(drake)` mechanism to find an installed instance of Drake.
 
-# Locally on Xenial
+# Instructions
 
-Download the binaries and install into /opt. This will ensure that CMake is able
-to auto-discover the `drake-config.cmake` module. 
+These instructions are only supported for Ubuntu 16.04 (Xenial).
 
 ```
-# A nightly build
+# Install PreRequisites
+<TODO: install pre-requisite debs list>
+
+# Install Drake to /opt/drake
 # curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-20170719-xenial.tar.gz
 # Alternatively, the latest (usually last night's build)
 curl -O https://s3.amazonaws.com/drake-packages/drake/nightly/drake-latest-xenial.tar.gz
 sudo tar -xvzf drake-latest-xenial.tar.gz -C /opt
-mkdir build
-cd build
+
+# Install GTest
+sudo apt-get install libgtest-dev
+cd /usr/src/gtest
+sudo mkdir build && cd build && sudo cmake .. && sudo make
+sudo cp *.a /usr/lib
```

Comment:
I went looking. There is no approved way (or at least I found none). I really don't like it, but I'm not gearing up for that battle!